From ddb9616d290ec5afc21753b39440d997c17ec2fe Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Tue, 2 May 2017 16:42:38 +0800
Subject: [PATCH] arm: dts: simplified rk3229 board configs.

Move EMMC and UART configs from every board DTS to a DTSI file.

Change-Id: If2fd49b9243b879ae89e172f55903eedfd4f3981
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 arch/arm/boot/dts/rk3229-echo-v10.dts | 14 --------------
 arch/arm/boot/dts/rk3229-evb.dts      |  7 +++----
 arch/arm/boot/dts/rk322x-android.dtsi | 16 +++++++++++++++-
 3 files changed, 18 insertions(+), 19 deletions(-)

diff --git a/arch/arm/boot/dts/rk3229-echo-v10.dts b/arch/arm/boot/dts/rk3229-echo-v10.dts
index 19bf62340576..484c1bda47e0 100644
--- a/arch/arm/boot/dts/rk3229-echo-v10.dts
+++ b/arch/arm/boot/dts/rk3229-echo-v10.dts
@@ -82,20 +82,6 @@
 	};
 };
 
-&emmc {
-	broken-cd;
-	bus-width = <8>;
-	cap-mmc-highspeed;
-	supports-emmc;
-	disable-wp;
-	non-removable;
-	num-slots = <1>;
-	/delete-property/ default-sample-phase;
-	/delete-property/ pinctrl-names;
-	/delete-property/ pinctrl-0;
-	status = "okay";
-};
-
 &i2c0 {
 	status = "okay";
 
diff --git a/arch/arm/boot/dts/rk3229-evb.dts b/arch/arm/boot/dts/rk3229-evb.dts
index 6905788e3db8..36ace42b66c5 100644
--- a/arch/arm/boot/dts/rk3229-evb.dts
+++ b/arch/arm/boot/dts/rk3229-evb.dts
@@ -1,4 +1,6 @@
 /*
+ * Copyright (c) 2017 Fuzhou Rockchip Electronics Co., Ltd
+ *
  * This file is dual-licensed: you can use it either under the terms
  * of the GPL or the X11 license, at your option. Note that this dual
  * licensing only applies to this file, and not this project as a
@@ -41,6 +43,7 @@
 /dts-v1/;
 
 #include "rk322x.dtsi"
+#include "rk322x-android.dtsi"
 
 / {
 	model = "Rockchip RK3229 Evaluation board";
@@ -111,7 +114,3 @@
 &u2phy1_host {
 	phy-supply = <&vcc_host>;
 };
-
-&uart2 {
-	status = "okay";
-};
diff --git a/arch/arm/boot/dts/rk322x-android.dtsi b/arch/arm/boot/dts/rk322x-android.dtsi
index 6729dbe7973a..54cc74a10801 100644
--- a/arch/arm/boot/dts/rk322x-android.dtsi
+++ b/arch/arm/boot/dts/rk322x-android.dtsi
@@ -16,7 +16,7 @@
  *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  *     GNU General Public License for more details.
  *
- * Or, alternatively,
+ *  Or, alternatively,
  *
  *  b) Permission is hereby granted, free of charge, to any person
  *     obtaining a copy of this software and associated documentation
@@ -80,6 +80,20 @@
 	enable-method = "psci";
 };
 
+&emmc {
+	broken-cd;
+	bus-width = <8>;
+	cap-mmc-highspeed;
+	supports-emmc;
+	disable-wp;
+	non-removable;
+	num-slots = <1>;
+	/delete-property/ default-sample-phase;
+	/delete-property/ pinctrl-names;
+	/delete-property/ pinctrl-0;
+	status = "okay";
+};
+
 &u2phy0 {
 	status = "okay";
 
-- 
2.35.3

